packages/bot-strategy-runner/README.md

```diff
@@ -0,0 +1,96 @@
+# @uma/bot-strategy-runner
```

Comment:
this looks really nasty like this in preview. take a look at the file [here](https://github.com/UMAprotocol/protocol/blob/51884a2382f26856f9d9beb31af5351897362eaa/packages/bot-strategy-runner/README.md) for something a bit easier to read.